docs/tutorials/ui-local-workflow.md

```diff
@@ -10,315 +10,298 @@ nav_order: 4
 
 ## Description
 
-Similar to the command line tool, Move2Kube Web-UI can also be used to do the translation. The Web-UI has all the capabilities that are there in the command line tool. This document explains the steps to bring up the UI and backend using docker-compose and use it for translation.
+Similar to the command line tool, Move2Kube Web-UI can also be used to do the transformation. The Web-UI has all the capabilities that are there in the command line tool. This document explains the steps to bring up the UI and backend using docker and use it for transformation.
 
 ## Prerequisites
 
-1. Install [Docker Engine](https://www.docker.com/get-started) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose).
+1. Install [Docker Engine](https://www.docker.com/get-started).
 
-2. Clone the [move2kube-ui](https://github.com/konveyor/move2kube-ui) repository
-
-   ```console
-   $ git clone https://github.com/konveyor/move2kube-ui.git
-
-   $ cd move2kube-ui
-   ```
-
-   Now, let's see the structure inside the directory.
-   ```console
-   move2kube-ui git:(main) $ ls
-   Dockerfile         README.md          contributing.md    package.json       tsconfig.json      webpack.prod.js
-   LICENSE            code-of-conduct.md docker-compose.yml server.js          webpack.common.js  wksps
-   Makefile           containers.txt     dr-surge.js        src                webpack.dev.js     yarn.lock
-   ```
-3. Download [language-platforms.zip](https://github.com/konveyor/move2kube-demos/raw/main/samples/language-platforms.zip) which we will be using for this tutorial. The language-platforms.zip file has a combination of multiple applications in different languages (Java, Go, Python, Ruby, etc.) which needs to be containerized and then put into Kubernetes.
+1. Download [language-platforms.zip](https://github.com/konveyor/move2kube-demos/raw/main/samples/language-platforms.zip) which we will be using for this tutorial. The language-platforms.zip file has a combination of multiple applications in different languages (Java, Go, Python, Ruby, etc.) which needs to be containerized and then put into Kubernetes.
 
 ## Steps to generate target artifacts
 
-1. Do a `docker-compose up`.
+1. Do a `docker run --rm -it -p 8080:8080 quay.io/konveyor/move2kube-ui`.
```

Comment:
Would be good to have the command with mounting

Reply:
done